<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LEInputFolder"/>
      </item>
      <item>
       <widget class="QPushButton" name="pBBrowseInputFolder">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LEOutputFolder"/>
      </item>
      <item>
       <widget class="QPushButton" name="pBBrowseOutPutFolder">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Numbers:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkB_1">
        <property name="text">
         <string>1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkB_6">
        <property name="text">
         <string>6</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkB_2">
        <property name="text">
         <string>2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkB_7">
        <property name="text">
         <string>7</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkB_3">
        <property name="text">
         <string>3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkB_8">
        <property name="text">
         <string>8</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkB_4">
        <property name="text">
         <string>4</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkB_9">
        <property name="text">
         <string>9</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkB_5">
        <property name="text">
         <string>5</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkB_0">
        <property name="text">
         <string>0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Count train data</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBTrainData">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Count test data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBTestData">
        <property name="maximum">
         <number>60000</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>152</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="pBGenerate">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_File_Close"/>
   </widget>
   <widget class="QMenu" name="mAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_About_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About_About"/>
    <addaction name="action_About_About_Qt"/>
    <addaction name="action_About_Site"/>
   </widget>
   <addaction name="mFile"/>
   <addaction name="mAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_About_Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_About_About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Search_Find">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="action_Search_Find_More">
   <property name="text">
    <string>Find more</string>
   </property>
  </action>
  <action name="action_Search_Find_and_replace">
   <property name="text">
    <string>Find / Replace</string>
   </property>
  </action>
  <action name="action_Settings_General">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="action_Settings_Module">
   <property name="text">
    <string>Module</string>
   </property>
  </action>
  <action name="action_Settings_Lan">
   <property name="text">
    <string>Lan</string>
   </property>
  </action>
  <action name="action_About_Site">
   <property name="text">
    <string>Site of program</string>
   </property>
  </action>
  <action name="action_Settings_Font">
   <property name="text">
    <string>Fonts</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="action_File_Close">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLang">
   <property name="text">
    <string>Lang</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian_2">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish_2">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFrance">
   <property name="text">
    <string>France</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="action_Settings_Language_Russian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="action_Settings_Language_France">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>France</string>
   </property>
  </action>
  <action name="action_Settings_Language_English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_Settings_Language_Deutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="action_About_About_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
